.travis.yml

```diff
@@ -33,7 +33,7 @@ install:
   - conda update conda
```

Comment:
change this to `conda install conda<4`.  That will fix this "helpful" behavior of conda 4.0.2 where it is doing things like attempting to remove databroker when you do `$ conda remove metadatastore`